ui/raidboss/data/07-dt/trial/valigarmanda-ex.txt

```diff
@@ -0,0 +1,170 @@
+# -p 9008:1006.7 95C3:2005.7 8FD1:3005.7 9007:4003.7
+# -it "Valigarmanda" "Flame-kissed Beacon"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+6.0 "--sync--" Ability { id: "8FF1", source: "Valigarmanda" }
+18.0 "Spikesicle" Ability { id: "8FF2|8FF4", source: "Valigarmanda" } duration 11.8
+# Too spammy to display each of these, and too close together to sync properly.
+26.0 "Sphere Shatter x10" #Ability { id: "995D", source: "Ice Boulder" } duration 10.8
+
+# Fire mini-phase
+# Skyruin has two separate casts, one for boss animation and one for actual damage.
+# The animation cast time ends while the on-screen cast bar is still running,
+# so sync to but don't display the first ability
+40.7 "--sync--" Ability { id: "95C4", source: "Valigarmanda" }
+46.1 "Skyruin (fire phase)" Ability { id: "8FD4", source: "Valigarmanda" }
+54.8 "Triscourge" Ability { id: "8FE7", source: "Valigarmanda" }
+62.9 "Volcanic Drop" Ability { id: "8FE5", source: "Valigarmanda" } duration 4.5
+76.3 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: "8FC8|8FCC|8FD0", source: "Valigarmanda" }
+# Partners
+77.1 "Charring Cataclysm" Ability { id: "8FC4", source: "Valigarmanda" }
+92.3 "Mountain Fire (cast)" Ability { id: "900C", source: "Valigarmanda" }
+# 9013/9014/9015 are boss positions for cleave
+# 9019 is the buster
+# 901A is the cleave
+# 900D-9012 seem to correspond to boss movement animation?
+# Only worry about the buster here, the rest make the mechanic too spammy
+# The sync time is tweaked to match the snapshot of the last cleave rather than the last buster, for safety
+97.8 "Mountain Fire 1" Ability { id: "9019", source: "Valigarmanda" }
+102.8 "Mountain Fire 2" Ability { id: "9019", source: "Valigarmanda" }
+107.8 "Mountain Fire 3" Ability { id: "9019", source: "Valigarmanda" }
+112.8 "Mountain Fire 4" Ability { id: "9019", source: "Valigarmanda" }
+117.8 "Mountain Fire 5" Ability { id: "9019", source: "Valigarmanda" }
+122.8 "Mountain Fire 6" Ability { id: "9019", source: "Valigarmanda" }
+# Light party stacks + puddles
+131.8 "Scourge of Fire 1" Ability { id: "8FEF", source: "Valigarmanda" }
+134.8 "Scourge of Fire 2" Ability { id: "8FEF", source: "Valigarmanda" }
+137.8 "Scourge of Fire 3" Ability { id: "8FEF", source: "Valigarmanda" }
+155.6 "Volcanic Drop" Ability { id: "8FE5", source: "Valigarmanda" }
+167.1 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: "8FC8|8FCC|8FD0", source: "Valigarmanda" }
+# Partners
+167.9 "Charring Cataclysm" Ability { id: "8FC4", source: "Valigarmanda" }
+# Spread
+177.9 "Scourge of Thunder" Ability { id: "8FEA", source: "Valigarmanda" }
+186.8 "Disaster Zone" Ability { id: "8FDA", source: "Valigarmanda" }
+# End fire mini-phase
+
+195.0 "Ruin Foretold" Ability { id: "9692", source: "Valigarmanda" }
+197.3 "--untargetable--"
+
+# Add phase
+198.3 "--targetable--"
```

Comment:
Could we move this below the block comments, or is there a reason to keep it separated?